### DISCUSSION TITLE
SSL Issue on Custom Domain - Need Assistance

### DISCUSSION THREAD

**Discussion by DjurovicAleksandar** [vercel/community#5820] [Help] [answered]
### Summary

Hello Vercel Community,

I hope this message finds you well. I am reaching out to seek help regarding an SSL issue I am facing on my website, konjicsummerfest.com, which is hosted on Vercel.

Here are the details:

Website: [konjicsummerfest.com](https://www.konjicsummerfest.com/)
Vercel Domain: [ksf-2024.vercel.app](https://ksf-2024.vercel.app/)

I am currently on the Hobby plan, and my website was functioning correctly for a few hours after deploying it on the custom domain konjicsummerfest.com. However, it suddenly stopped working, and when I try to access it, I receive a message indicating that the site is not secure.

Upon investigating with the organization where I purchased the domain, they advised me to reach out to the Vercel team for assistance with the SSL issue.

I would greatly appreciate it if someone could guide me on how to resolve this SSL issue for my custom domain konjicsummerfest.com. Additionally, I would like to understand how to apply SSL to this domain from Vercel.

Thank you in advance for your assistance!

Best regards,
Aleksandar

### Example

https://www.konjicsummerfest.com/

### Steps to Reproduce

https://github.com/vercel/community/assets/117927834/8e814c9f-703f-4113-bf53-163ae5ff5f59

Here is written: 
website is not available
Unable to find the DNS address of the host computer konjic summer fest.com. Diagnosing the problem

**Comment by amyegan** [comment, marked as answer]
Hi @DjurovicAleksandar. I tried just now and was able to access the site without any errors. Are you still experiencing this problem?

**Reply by DjurovicAleksandar** [reply]
Hey, after few days, the site started working as usual. I guess the issue was with the dns propagination. Thank you
